assessment.md

```diff
@@ -13,81 +13,512 @@
 
 ### Group
 
 * Attendance
+  * N = Missing five or more lab sessions
+  * I = Missing four lab sessions
+  * A = Missing three lab sessions
+  * G = Missing two lab sessions
+  * E = Missing one or less lab sessions
 * Deadlines
+  * N = If an assignment is turned in 5 or more days past the deadline
+  * I = If an assignment is turned in 1-4 days past the deadline
+  * A = If an assignment is turned in on the day of, or before the deadline
+  * G = If an assignment is turned in on the day of, or before the deadline
+  * E = If an assignment is turned in on the day of, or before the deadline
 * Communication
+  * N = Never check appropriate slack channels and never communicate with
+   teammates
+  * I = Rarely check appropriate slack channels and rarely communicate with
+    teammates
+  * A = Sometimes check appropriate slack channels and sometimes communicate
+    with teammates
+  * G = Regularly check appropriate slack channels and regularly communicate
+    with teammates
+  * E = Always check appropriate slack channels and always communicate
+    effectively with teammates
 * Participation
+  * N = Never participate in group conversation or activity
+  * I = Rarely participate in group conversation or activity
+  * A = Sometimes participate in group conversation or activity
+  * G = Regularly participate in group conversation or activity
+  * E = Always participate in group conversation or activity
 * Conduct
+  * N = Disrespectful and excludes teammates
+  * I = Rarely respectful and inclusive of teammates
+  * A = Sometimes respectful and inclusive of teammates
+  * G = Regularly respectful and inclusive of teammates
+  * E = Always respectful and inclusive of teammates
 * Decisions
+  * N = Never participate in the decision making process and share your opinion.
+  * I = Rarely participate in the decision making process and share your opinion
+  * A = Sometimes participate in the decision making process and share your opinion
+  * G = Regularly participate in the decision making process and share your opinion
+  * E = Always participate in the decision making process and share your opinion.
 * Postmortems
+  * N = Never participate the analysis or discussion after the work is done
+  * I = Rarely participate the analysis or discussion after the work is done
+  * A = Sometimes participate the analysis or discussion after the work is done
+  * G = Regularly participate the analysis or discussion after the work is done
+  * E = Always participate the analysis or discussion after the work is done
 * Conflicts
+  * N = Never handles differences with teammates in mature manner
+  * I = Rarely handles differences with teammates in mature manner
+  * A = Sometimes handles differences with teammates in mature manner
+  * G = Regularly handles differences with teammates in mature manner
+  * E = Always handles differences with teammates in mature manner
 * Infractions
+  * N = Always breaks the team rules
```

Comment:
This section may not be needed when comparing with the conduct repository's work.